pam-u2f.c

```diff
@@ -278,6 +282,38 @@ int pam_sm_authenticate(pam_handle_t *pamh, int flags, int argc,
     }
   }
 
+  int should_free_authpending_file = 0;
+  // Determine the full path for authpending_file in order to emit touch request notifications
+  if (!cfg->authpending_file) {
+    size_t actual_size = snprintf(buffer, BUFSIZE, DEFAULT_AUTHPENDING_FILE_PATH, getuid());
```

Comment:
snprintf returns an int, size_t is an unsigned type meaning it'll never be less than 0